modules/core/arrow-instances/src/main/kotlin/arrow/instances/coproduct.kt

```diff
@@ -54,8 +54,8 @@ interface CoproductTraverseInstance<F, G> : Traverse<CoproductPartialOf<F, G>> {
 
     fun TG(): Traverse<G>
 
-    override fun <H, A, B> traverse(fa: CoproductOf<F, G, A>, f: (A) -> Kind<H, B>, GA: Applicative<H>): Kind<H, Coproduct<F, G, B>> =
-            fa.fix().traverse(f, GA, TF(), TG())
+    override fun <H, A, B> Applicative<H>.traverse(fa: CoproductOf<F, G, A>, f: (A) -> Kind<H, B>): Kind<H, Coproduct<F, G, B>> =
```

Comment:
can we also add `Kind<F, A>.sequence() = traverse(::identity, GApplicative)` if it isn't already defined as syntax for all Traverse instances target data types?

Reply:
It's there already

Reply:
Oh wait.